index.html

```diff
@@ -9,5 +9,14 @@
   <link rel="stylesheet"  type="text/css" href="application.css">
 </head>
 <body>
+    <div class="body-container">
```

Comment:
Indentation should be 2 spaces (1 tabulation)
For example:
.You have:
<img width="643" alt="captura de pantalla 2018-05-30 a la s 17 08 18" src="https://user-images.githubusercontent.com/36163179/40747766-47d6f6fc-642c-11e8-8b97-92f097c283f5.png">

.It should look like this:
<img width="581" alt="captura de pantalla 2018-05-30 a la s 17 09 08" src="https://user-images.githubusercontent.com/36163179/40747769-4b3cbda4-642c-11e8-9bea-891620bc1dba.png">


`Indentation should be of 2 spaces each`